### ISSUE TITLE
Any chance of a Firefox version?

### ISSUE THREAD

**Issue by arogl** [Balthier/Enhanced-Plex#76]
I did use the mozilla tool to try and see what needed to be fixed, I followed some online links to attempt to fix the manifest file, but JS is not in my skillset.

[Validation Results for enhanced_plex.crx.zip](https://github.com/user-attachments/files/17553161/Validation.Results.for.enhanced_plex.crx.zip)

**Comment by Balthier** [comment]
This is something that I looked at very early on, but ran into a few complications. Now everything is more stable, I'll take another look!

**Comment by Balthier** [comment]
Just an FYI - I've finally got round to making this a FF extension. It's currently in the process of being tested, but should appear with v4.0.0

**Comment by arogl** [comment]
happy to help test

**Comment by Balthier** [comment]
I've tentatively uploaded the 4.0.0 version. Feel free to have a go: https://addons.mozilla.org/en-US/firefox/addon/enhanced-plex/

**Comment by arogl** [comment]
I have had a play around on Firefox and it all seems to be working.

I did notice that the colours for charts are different. I tried to review the html, but the only thing I could see was the CSS was not availabale in the FireFox version under the view-source.

I verified the CSS was in the `.xpi` downloaded.

Chrome

<img width="1787" height="691" alt="Image" src="https://github.com/user-attachments/assets/aeafc05c-ec57-4677-8afd-92a7e2aaa31a" />

Firefox

<img width="1793" height="696" alt="Image" src="https://github.com/user-attachments/assets/96435a63-2cb8-4ce9-92e6-1fe926871005" />

**Comment by Balthier** [comment]
Yeah, I expect some colours may be different - There's 2 different colour "schemes". For simplicity sake, for most charts, the colours are essentially randomly generated upon first load. However, on some of the bar charts, I used an actual scale with Green being the highest count, and red being the lowest

One thing I am noticing in both though, is the bar charts are not sorted High->Low - I thought I did that a while ago. Might need to take a look at that again
